app.js

```diff
@@ -0,0 +1,35 @@
+require("dotenv").config();
+
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const routes = require("./routes");
+const { appDataSource } = require("./models/data-source");
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("combined"));
+
+app.use(routes);
+
+app.get("/ping", (req, res) => {
+  res.json({ message: "pong" });
+});
```

Comment:
```suggestion
app.get("/ping", (req, res) => {
  res.status(200).json({ message: "pong" });
});
```
status 코드도 지정해서 작성될 수 있게 수정해주세요